src/components/Alerts/src/alert.css

```diff
@@ -0,0 +1,151 @@
+.denhaag-alert {
+  border-width: 0;
+  border-radius: var(--denhaag-alert-border-radius);
+  color: var(--denhaag-alert-text-color);
```

Comment:
This token isn't defined anywhere by default

Reply:
Removed it